src/ripple/overlay/ReduceRelayCommon.h

```diff
@@ -17,22 +17,31 @@
 */
 //==============================================================================
 
-#ifndef RIPPLE_OVERLAY_SQUELCHCOMMON_H_INCLUDED
-#define RIPPLE_OVERLAY_SQUELCHCOMMON_H_INCLUDED
+#ifndef RIPPLE_OVERLAY_REDUCERELAYCOMMON_H_INCLUDED
+#define RIPPLE_OVERLAY_REDUCERELAYCOMMON_H_INCLUDED
+
 #include <chrono>
+#include <stdlib.h>
 
 namespace ripple {
 
-namespace squelch {
-
-using namespace std::chrono;
+namespace reduce_relay {
 
 // Peer's squelch is limited in time to
-// rand{MIN_UNSQUELCH_EXPIRE, MAX_UNSQUELCH_EXPIRE}
-static constexpr seconds MIN_UNSQUELCH_EXPIRE = seconds{300};
-static constexpr seconds MAX_UNSQUELCH_EXPIRE = seconds{600};
+// rand{MIN_UNSQUELCH_EXPIRE, max_squelch},
+// where max_squelch is
+// min(max(MAX_UNSQUELCH_EXPIRE_DEFAULT, SQUELCH_PER_PEER * number_of_peers),
+//     MAX_UNSQUELCH_EXPIRE_PEERS)
+static constexpr std::chrono::seconds MIN_UNSQUELCH_EXPIRE =
```

Comment:
These could be `static constexrp auto` but no big deal.

Reply:
Updated.